erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -1239,11 +1239,18 @@ def set_missing_values(source, target):
 		target.discount_amount = 0.0
 		target.inter_company_order_reference = ""
 		target.shipping_rule = ""
+		target.tc_name = ""
+		target.terms = ""
+		target.payment_terms_template = ""
```

Comment:
I think the `payment_schedule` should be cleared. Only clearing the `payment_terms_template` didn't work for me. The `payment_schedule` still got copied.

Reply:
Only works properly in version 15. I think the issue will be in version 14, but I fixed it.